api/v1alpha3/types.go

```diff
@@ -179,7 +179,14 @@ type NetworkSpec struct {
 	// Subnets configuration.
 	// +optional
 	Subnets Subnets `json:"subnets,omitempty"`
-}
+
+	// TagSecurityGroups optionally disables tagging the security groups used by the cluster
+	// +optional
+	TagSecurityGroups *bool `json:"tagSecurityGroups,omitempty"`
```

Comment:
Given that we are going to tag security groups that we create, and I would assume provided security groups would have separate lifecycle management, do we actually need to worry about exposing the ability to disable tagging here?

Reply:
Sounds good to me! Will update.

Reply:
sgtm, thanks!